tests/garage/torch/modules/test_gaussian_mlp_module.py

```diff
@@ -0,0 +1,368 @@
+import pickle
+from garage.torch.modules.gaussian_mlp_module import GaussianMLPModule
+import unittest
+from unittest import mock
+import torch
+from torch import nn
+from nose2.tools.params import params
+
+
+class TestGaussianMLPmodule(unittest.TestCase):
+
+    @mock.patch('torch.rand')
+    @params((5, 1, (1,)), (5, 1, (2,)), (5, 2, (3,)), (5, 2, (1, 1)), (5, 3, (2, 2)))
```

Comment:
for parameterized tests, please format these to have one set of arguments (tuple) per line. you may need to use `# yapf: disable`/`# yapf: enable`

Reply:
fixed